### ISSUE TITLE
Configure severity of non existing package

### ISSUE THREAD

**Issue by laeubi** [eclipse-pde/eclipse.pde#502]
I have a bundle that intentionally contains a package export that is (not yet because generated as part of the build) there, and when PDE complains about this it offers to remove or configure the severity:

![grafik](https://user-images.githubusercontent.com/1331477/224469160-30db19fd-6140-46ee-a83a-f21aa7403bd7.png)

then I choose to configure the project and get this screen but can't configure the severity for non existing packages:

![grafik](https://user-images.githubusercontent.com/1331477/224469251-6d0d7652-3e17-4e0f-affa-0beef575da72.png)

**Comment by vik-chand** [comment]
<img width="708" alt="image" src="https://user-images.githubusercontent.com/99390647/224655520-e72742ac-0236-4dee-b151-3a7f893bc549.png">
This is controlled by this particular option which sounds OK to me

**Comment by laeubi** [comment]
> This is controlled by this particular option which sounds OK to me 

But a package export is not a dependency, and I would assume this will also silence missing package imports right?

**Comment by vik-chand** [comment]
> > This is controlled by this particular option which sounds OK to me
> 
> But a package export is not a dependency, and I would assume this will also silence missing package imports right?

That's true. We could probably rename this preference to indicate that. I dont think 1 more preference  is warranted just for this ( though it can be an option). Ideally if we had a problem filter ( like the API tool filter), that would have worked too. We could also change it to INFO ( to not completely silence and woraround this scenario).
